packages/flutter/test/widgets/shortcuts_test.dart

```diff
@@ -257,7 +255,6 @@ void main() {
                 TestAction.key: () => TestAction(
                   onInvoke: (FocusNode node, Intent intent) {
                     invoked = true;
-                    return true;
                   },
```

Comment:
@gspencergoog FYI (in case you were expecting these return values to have any effect)

Reply:
No, that's just a mistake caught by the lint.